data/human/human missions.txt

```diff
@@ -5547,3 +5547,221 @@ mission "Quicksilver Mixup 1"
+	source Martini
+	destination Chiron
+	to offer
+		has "main plot completed"
+		random < 15
+	on offer
+		conversation
+			`As you walk through <origin>'s cramped new planetary spaceport, you see a middle-aged man on the other side of the corridor, behind a column of people. He's just sitting on a bench looking ahead at nowhere in particular, with a suitcase to his side, and holding a cane in one hand. Nobody's paying any attention to him, but he is tapping his cane every so often someone passes by him.`
+			choice
+				`	(Go up to him.)`
+				`	(Ignore him.)`
+					decline
+			`	Wading through the mix of passengers, stevedores and captains in the overflowing corridor, you reach the man with the cane. It's obvious now that he's blind, as he doesn't move to look at you and continues looking straight ahead, but he taps his cane twice when you come by him.`
+			choice
+				`	"Hello."`
+				`	"Are you alright?"`
+				`	(Nevermind.)`
+					decline
+			`	"Yes, hello, how are you doing?" the man responds, shifting his head towards your direction. "I'm here waiting for someone to pick me up," he says in a voice that sounds like he expects it at any moment.`
+
+			label questions
+			choice
+				`	"Are they far away?"`
+					goto far
+					to display
+						not "Blind Man from Martini: label far"
+				`	"Who is it you're looking for?"`
+					goto looking
+					to display
+						has "Blind Man from Martini: label far"
+						not "Blind Man from Martini: label looking"
+				`	"Where is she?"`
+					goto somewhere
+					to display
+						has "Blind Man from Martini: label looking"
+				`	(Nevermind, he's just a blind man.)`
+					decline
+			
+			label far
+			action
+				set "Blind Man from Martini: label far"
+			`	"Well, I suppose you could say that they are far away, or they could be right here. I don't know, I can't see them." He taps his cane twice more.`
+				goto questions
+			
+			label looking
+			action
+				set "Blind Man from Martini: label looking"
+			`	"Someone to take me to my daughter, Anita." Yet again, his cane taps twice.`
+				goto questions
+
+			label somewhere
+			`	He looks over to his side, as if he feels reserved about something. "She's on <planet>, getting some experimental treatment from the University of New Sydney. I want to be by her side if anything goes wrong, to tell her that I'm there for her, that I love her, and that everything will be okay. She's been waiting for the treatment for so long now. And since nobody else seems to have arrived here yet, I hope you'll be the one to take me there."`
+			choice
+				`	"Of course, I'll take you to <planet>."`
+				`	"Sorry, I won't be able to."`
+					goto can't
+			`	The man musters a broad smile, says "Thank you all too much. Could you help me with the luggage? Of course, I can't see where it is..." and scans the floor with his cane for it. Later, while boarding, he tells you his name is Douglas, and he promises to you <payment> for transporting him.`
```

Comment:
```suggestion
			`	The man musters a broad smile, says "Thank you all too much. Could you help me with the luggage? Of course, I can't see where it is..." and scans the floor with his cane for it. Later, while boarding, he tells you his name is Douglas Pears, and he promises to you <payment> for transporting him.`
```
Need the last name because it isn't mentioned before `<first>` mentions it at the hospital. I'd suggest either adding an option for the player to ask about payment before agreeing (or just have Douglas mention it before the player decides).